.github/workflows/codeQL-analysis.yaml

```
@@ -0,0 +1,24 @@
+---
+name: Static analysis with CodeQL
```

Let's keep name shorts

* rename file to `codeql.yaml`
and: 
```suggestion
name: codeql
```

---

Done